model_compression_toolkit/qat/keras/quantizer/quant_utils.py

```diff
@@ -17,6 +17,23 @@
 from typing import Tuple
 
 
+def ste_round(x: tf.Tensor) -> tf.Tensor:
+    """
+    Return the rounded values of a tensor.
+    """
+    error = tf.stop_gradient(tf.math.round(x) - x)
+    return error + x
+
+
+def ste_scale(x: tf.Tensor, scale=1.0) -> tf.Tensor:
```

Comment:
why do we need STE for scaling?

Reply:
This is one of the component of LSQ: STE for the scale so it will influence only on backward pass)